component/src/test/java/org/wso2/extension/siddhi/io/http/source/HttpsSourceTestCaseForSSL.java

```diff
@@ -268,36 +232,108 @@ public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
         // publishing events
         List<String> expected = new ArrayList<>(2);
-        String event1 =
-                "<events><event><name>John</name>" + "<age>100</age><country>Sri Lanka</country></event></events>";
-        String event2 = "<events><event><name>Mike</name>" + "<age>20</age><country>USA</country></event></events>";
-        try {
-            new HttpTestUtil().httpsPublishEvent(event1, "https://localhost:9095/endpoints/RecPro", false,
+        String event1 = "<events>"
+                            + "<event>"
+                                + "<name>John</name>"
+                                + "<age>100</age>"
+                                + "<country>AUS</country>"
+                            + "</event>"
+                        + "</events>";
+        String event2 = "<events>"
+                            + "<event>"
+                                + "<name>Mike</name>"
+                                + "<age>20</age>"
+                                + "<country>USA</country>"
+                            + "</event>"
+                        + "</events>";
+        new HttpTestUtil().httpsPublishEvent(event1, "https://localhost:8005/endpoints/RecPro", false,
                     "text/plain");
-            new HttpTestUtil().httpsPublishEvent(event2, "https://localhost:9095/endpoints/RecPro", false,
+        new HttpTestUtil().httpsPublishEvent(event2, "https://localhost:8005/endpoints/RecPro", false,
                     "text/plain");
-        } catch (IllegalArgumentException e) {
-            e.getClass().getCanonicalName();
-        }
-        final TestAppender appender = new TestAppender();
-        final Logger logger = Logger.getRootLogger();
-        logger.addAppender(appender);
-        try {
-            Logger.getLogger(HttpsSourceTestCaseForSSL.class).error("Test");
-        } finally {
-            logger.removeAppender(appender);
-        }
         final List<LoggingEvent> log = appender.getLog();
-        final LoggingEvent firstLogEntry = log.get(0);
-        org.hamcrest.MatcherAssert.assertThat(firstLogEntry.getLevel(), is(Level.ERROR));
-        org.hamcrest.MatcherAssert.assertThat(firstLogEntry.getMessage(), is("Test"));
-        org.hamcrest.MatcherAssert.assertThat(firstLogEntry.getLoggerName(), is("org.wso2.extension.siddhi.io" +
-                ".http.source.HttpsSourceTestCaseForSSL"));
+        org.hamcrest.MatcherAssert.assertThat(log.get(3).getLevel(), is(Level.ERROR));
+        org.hamcrest.MatcherAssert.assertThat(log.get(4).getLevel(), is(Level.ERROR));
+        org.hamcrest.MatcherAssert.assertThat(log.get(3).getLoggerName(), is("org.wso2.extension.siddhi.io" +
+                ".http.source.util.HttpTestUtil"));
         Thread.sleep(100);
         Assert.assertEquals(receivedEventNameList.toString(), expected.toString());
         executionPlanRuntime.shutdown();
     }
-    class TestAppender extends AppenderSkeleton {
+
+//    /**
```

Comment:
Let's remove commented out code

Reply:
Fixed in 602e96bfa91ea786afe9f9bd414b1bc8b1787631